Library/Homebrew/test/cask/installer_spec.rb

```diff
@@ -6,6 +6,8 @@
     let(:empty_depends_on_stub) {
       double(formula: [], cask: [], macos: nil, arch: nil, x11: nil)
     }
+    let(:diskutil_list_command) { "diskutil list | grep '/dev'" }
+    let(:original_diskutil_list) { `#{diskutil_list_command}` }
```

Comment:
`let` is lazily evaluated, use `let!`.

Reply:
I ended up rewriting test again since the `let` should only run for the DMG tests.

I had moved them back into `it` block and ended up having to add a few more sleeps due to still seeing some `diskutil eject/unmount` timing issues on some random seeds.
Sadly a bit of a hack, but I don't know if there is a better way.

Please re-review when you have the chance. Thanks.